src/middlewared/middlewared/alembic/versions/25.04/2025-02-19_18-47_remove_blacklisted_aux_params.py

```diff
@@ -0,0 +1,48 @@
+"""Remove blacklisted SMB share aux params
+
+Revision ID: ae78ab5ebf07
+Revises: d93139a68db5
+Create Date: 2025-02-19 18:47:53.749089+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ae78ab5ebf07'
+down_revision = 'd93139a68db5'
+branch_labels = None
+depends_on = None
+
+
+SHARE_BLACKLIST = (
+    'wide links',
+    'use sendfile',
+    'vfs objects',
+    'insecure',
```

Comment:
I'm not familiar with `insecure`.

Reply:
variations of "insecure wide links"

Reply:
Isn't that: "allow insecure wide links"

The test is only for `startswith`